lib/ios/AirMaps/AIRMapManager.m

```diff
@@ -599,6 +599,8 @@ - (UIView *)view
                             @"countryCode" : [NSString stringWithFormat:@"%@", placemark.ISOcountryCode],
                             @"country" : [NSString stringWithFormat:@"%@", placemark.country],
                         });
+                    } else {
+                        reject(@"Invalid argument", [NSString stringWithFormat:@"Can not get address location"], NULL);
```

Comment:
Can we update the message here to match android? “ Can not get address location”

Reply:
Both android and iOS have the same `error.message` and that is `Can not get address location`, however it seems that through this library only iOS errors have an `error.code` in this case is `Invalid argument` which may not suit.

Reply:
For reference both of these errors will look like this on the JS side.

android
```
{
  "code": "EUNSPECIFIED",
  "message": "Can not get address location",
  "userInfo": null,
  "nativeStackAndroid": []
}
```
ios
```
{
  "code": "Invalid argument",
  "message": "Can not get address location",
  "domain": "RCTErrorDomain",
  "userInfo": null,
  "nativeStackIOS": [
    "0   Wayke                               0x000000010115e9f0 RCTJSErrorFromCodeMessageAndNSError + 120",
    "1   Wayke                               0x00000001010f5a08 __41-[RCTModuleMethod processMethodSignature]_block_invoke_2.129 + 152",
    "2   Wayke                               0x00000001015d9aa0 __72-[AIRMapManager getAddressFromCoordinates:coordinate:resolver:rejecter:]_block_invoke_2 + 188",
    "3   libdispatch.dylib                   0x0000000180100580 _dispatch_call_block_and_release + 24",
    "4   libdispatch.dylib                   0x0000000180101d44 _dispatch_client_callout + 16",
    "5   libdispatch.dylib                   0x000000018010f2f4 _dispatch_main_queue_callback_4CF + 956",
    "6   CoreFoundation                      0x0000000180365d1c __CFRUNLOOP_IS_SERVICING_THE_MAIN_DISPATCH_QUEUE__ + 12",
    "7   CoreFoundation                      0x000000018036025c __CFRunLoopRun + 2448",
    "8   CoreFoundation                      0x000000018035f3bc CFRunLoopRunSpecific + 572",
    "9   GraphicsServices                    0x000000018afdd70c GSEventRunModal + 160",
    "10  UIKitCore                           0x00000001843f03d0 -[UIApplication _run] + 964",
    "11  UIKitCore                           0x00000001843f51ac UIApplicationMain + 112",
    "12  Wayke                               0x00000001007f0a10 main + 108",
    "13  libdyld.dylib                       0x0000000180224554 start + 4"
  ]
}
```